src/use_case/search/SearchDataAccessInterface.java

```diff
@@ -18,7 +18,7 @@ public interface SearchDataAccessInterface {
 
     List<String> getStationAmenities(String inputStationName);
 
-    void setStation(Station stationObj);
+    void setStation(String stationName);
     void setStationAmenities(Station stationObj);
```

Comment:
should we fix the parameter in setStationAmenities as String stationName or keep it the same